test/builtin/drawing/svg_outline.py

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/env python3
```

Comment:
I would prefer to build on an existing SVG formatting tool than add one to this code base.